bee-api/bee-rest-api/src/endpoints/routes/api/v2/milestone_by_milestone_index.rs

```diff
@@ -0,0 +1,70 @@
+// Copyright 2020-2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use std::net::IpAddr;
+
+use bee_message::payload::{
+    milestone::{MilestoneId, MilestoneIndex},
+    MilestonePayload,
+};
+use bee_runtime::resource::ResourceHandle;
+use bee_storage::access::Fetch;
+use bee_tangle::milestone_metadata::MilestoneMetadata;
+use log::error;
+use warp::{filters::BoxedFilter, reject, Filter, Rejection, Reply};
+
+use crate::{
+    endpoints::{
+        config::ROUTE_MILESTONE, filters::with_storage, path_params::milestone_index, permission::has_permission,
+        rejection::CustomRejection, storage::StorageBackend,
+    },
+    types::responses::MilestoneResponse,
+};
+
+fn path() -> impl Filter<Extract = (MilestoneIndex,), Error = Rejection> + Clone {
+    super::path()
+        .and(warp::path("milestones"))
```

Comment:
Route should be `/api/v2/milestones/index/{}` ?

Reply:
"by-index", fixed it